redfish-core/lib/log_services.hpp

```diff
@@ -4154,6 +4154,85 @@ inline void getSystemHardwareIsolationLogEntryById(
         objMapperIface, "GetObject", entryObjPath.str, hwIsolationEntryIfaces);
 }
 
+/**
+ * @brief API Used to deisolate the given HardwareIsolation entry.
+ *
+ * @param[in] req - The HardwareIsolation redfish request (unused now).
+ * @param[in] asyncResp - The redfish response to return.
+ * @param[in] entryId - The entry id of HardwareIsolation entries to deisolate.
+ *
+ * @return The redfish response in the given buffer.
+ */
+inline void deleteSystemHardwareIsolationLogEntryById(
+    const crow::Request& /* req */,
```

Comment:
what is the need to pass this 'req' if its not used ? Similar comment in multiple places